#!/usr/bin/env python
PACKAGE = "masterslave"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
translationalGains = gen.add_group("Translational Gains")
translationalGains.add("PTrans", double_t, 0, "Gain of the propotional translation part", 0, 0.0, 1)
translationalGains.add("DTrans", double_t, 0, "Gain of the derivational translation part", 0, 0.0, 1)


rotationalGains = gen.add_group("Rotational Gains")
rotationalGains.add("PRot", double_t, 0, "Gain of the propotional rotation part", 0, 0.0, 1)
rotationalGains.add("DRot", double_t, 0, "Gain of the derivational rotation part", 0, 0.0, 1)

gen.add("ResetInit", bool_t, 0, "Reset Initial Pose", False)

exit(gen.generate(PACKAGE, "VisualServoing", "VisualServoing"))
